README.md

```diff
@@ -22,173 +18,177 @@ Currently, our SDK supports interactions with our application-specific rollup ba
-    l1Signer,
-    l2Signer,
-  };
-};
+// Create signers
+const ethWallet = new Wallet(YOUR_PRIVATE_ETH_KEY);
+const ethSigner = ethWallet.connect(provider);
+const starkSigner = Utils.createStarkSigner(YOUR_PRIVATE_STARK_KEY);
 ```
 
-### Standard API Requests
+> **Warning**
+> If you generate your own signers, you will have to persist the Stark private key. The key ***cannot*** be deterministically retrieved using the Wallet SDK.
 
-The Core SDK includes classes that interact with the Immutable X APIs.
+### Examples
 
-```ts
-// Standard API request example usage
-import { getConfig, AssetsApi } from '@imtbl/core-sdk';
-
-const getYourAsset = async (tokenAddress: string, tokenId: string) => {
-  const config = getConfig({
-    coreContractAddress: '0x4527BE8f31E2ebFbEF4fCADDb5a17447B27d2aef',
-    registrationContractAddress: '0x6C21EC8DE44AE44D0992ec3e2d9f1aBb6207D864',
-    chainID: 3,
-    basePath: 'https://api.ropsten.x.immutable.com',
-  });
-  const assetsApi = new AssetsApi(config.apiConfiguration);
-
-  const response = await assetsApi.getAsset({
-    tokenAddress,
-    tokenId,
-  });
-
-  return response;
-};
-```
+#### Create a project (requires an Ethereum layer 1 signer)
 
-View the [OpenAPI spec](openapi.json) for a full list of API requests available in the Core SDK.
+```ts
+const createProjectResponse = await client.createProject(ethSigner, {
+  company_name: 'My Company',
+  contact_email: 'project@company.com',
+  name: 'Project name',
+});
 
-### Authorised project owner requests
+const projectId = createProjectResponse.data.id.toString();
 
-Some methods require authorisation by the project owner, which consists of a Unix epoch timestamp signed with your ETH key and included in the request header.
+const getProjectResponse = await client.getProject(ethSigner, projectId);
+```
 
-On project and collection methods that require authorisation, this signed timestamp string can typically be passed as the `iMXSignature` and `iMXTimestamp` parameters.
+#### Deposit tokens from L1 to L2 (requires an Ethereum layer 1 signer)
 
 ```ts
-// Example method to generate authorisation headers
-const getProjectOwnerAuthorisationHeaders = async (signer: Signer) => {
-  const timestamp = Math.floor(Date.now() / 1000).toString();
-  const signature = await signRaw(timestamp, signer);
-
-  return {
-    timestamp,
-    signature,
-  };
-};
-
-// Using generated authorisation headers
-const createProject = async (
-  name: string,
-  company_name: string,
-  contact_email: string,
-) => {
-  const api = new ProjectsApi(this.config.api);
-  const { timestamp, signature } = getProjectOwnerAuthorisationHeaders(signer);
-
-  return await api.createProject({
-    createProjectRequest: {
-      name,
-      company_name,
-      contact_email,
-    },
-    iMXSignature: signature,
-    iMXTimestamp: timestamp,
-  });
-};
+const depositResponse = await client.deposit(ethSigner, {
+  type: 'ETH',
+  amount: '500000000000000000', // Amount in wei
```

Comment:
I think we should capitalise all references of `Wei`, as it is a unit.

Reply:
@sprngbk

Reply:
Discussed offline, lowercase seems to be the standard capitalization. No longer needed.